### ISSUE TITLE
SkyFloatingLabelTextField.framework: required code signature missing

### ISSUE THREAD

**Issue by bludginozzie** [Skyscanner/SkyFloatingLabelTextField#201]
I am unsure if this is a bug or local issue.  I am positing it here to see if anyone else is experiencing the same problem.

I had a fully working project with no problems.  I then recently upgraded to the latest version of SkyFloatingLabelTextField (v3.4.1) and also upgraded Cocoapods (v1.5.0) and I am now getting a strange error in the storyboard editor.

> file:///..../Base.lproj/Main.storyboard: error: IB Designables: Failed to render and update auto layout status for LoginViewController (WxM-Yi-A5J): dlopen(SkyFloatingLabelTextField.framework, 1): no suitable image found.  Did find:
> 	SkyFloatingLabelTextField.framework: required code signature missing for 'SkyFloatingLabelTextField.framework'

I have tried uninstalling the SkyFloatingLabelTextField cocoapod and then reinstalling it with no change

The app runs fine in the simulator, the storyboard editor is unusable since it renders the SkyFloatingLabelTextField in strange positions.

Xcode Version: 9.3
SkyFloatingLabelTextField Version: 3.4.1
Deployment Target:  11.2
Base SDK: 11.3
Method of Integration: CocoaPods

Any advice would be greatly appreciated.

**Comment by musbaalbaki** [comment]
+1

**Comment by xRena** [comment]
+1

**Comment by dcw008** [comment]
+1

**Comment by eitanaviv** [comment]
+1

**Comment by k0nserv** [comment]
Hey everyone. That doesn't sound too good. Does reverting to 3.4.0 solve the issue for the time being. I'll have a look at what changed between 3.4.1 and 3.4.0, but my hunch is that nothing changed which would cause this.

**Comment by k0nserv** [comment]
I had a look and there's nothing in 3.4.1 that I think would have caused this. Can everyone please try this https://stackoverflow.com/questions/40110392/required-code-signature-missing-for-a-library. Also please refrain from +1 comments as they make the thread hard to read, use reactions instead.

**Comment by ahmedhatem003** [comment]
Thank you @k0nserv 
I'll use reactions next time
I went through solutions in the link but couldn't find _CodeSignature folder in SkyFloatingLabelTextField.framework in finder
<img width="626" alt="screen shot 2018-04-16 at 17 52 13" src="https://user-images.githubusercontent.com/20728502/38820981-96530ff0-419f-11e8-85c0-2fe05e084423.png">

**Comment by bludginozzie** [comment]
No, rolling back to 3.4.0 does not fix the issue.  Could it be an incompatibility with Cocoapods 1.5? I noticed it has some updates for static libraries that maybe causing this issue? http://blog.cocoapods.org/CocoaPods-1.5.0/

**Comment by k0nserv** [comment]
@bludginozzie That's a possibility for sure. I'll have to have a look and see if there are any actions for library authors under 1.5.0

**Comment by bludginozzie** [comment]
Thank you, the workaround works great.

**Comment by merlin910** [comment]
I also have this same issue when integrating via Carthage the log file points to the .swiftlint.yml file and when I try to build manually I have to comment out that script to get it to build.

**Comment by k0nserv** [comment]
@merlin910 I'm not quite sure how to disable custom scripts when Carthage is used.

**Comment by HariNarayanan777** [comment]
the error was gone after i paste the below code in my pod file. and install the pod once again.

> # Workaround for Cocoapods issue #7606
post_install do |installer|
    installer.pods_project.build_configurations.each do |config|
        config.build_settings.delete('CODE_SIGNING_ALLOWED')
        config.build_settings.delete('CODE_SIGNING_REQUIRED')
    end
end
